OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -463,6 +462,24 @@ public interface IBotPositionsUpdated
 		void UpdatedDefenseCenter(CPos newLocation);
 	}
 
+	[RequireExplicitImplementation]
+	public interface IBotIdleBaseUnits
+	{
+		void UpdatedUnitsHangingAroundBaseCount(int newCount);
```

Comment:
Reporting the number of idle units without having a way to see what they are doesn't make sense for any reasonable future implementations.

How about we change this to `IBotNotifyIdleBaseUnits` and passing the actual list instead?